OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -0,0 +1,308 @@
+# Oracle WebCenter Portal 12.2.1.4.0 on Docker
+
+Before running an Oracle WebCenter Portal image below security configuration are recommended to be followed.
+## Docker Security Configuration
+
+### A. Configuring auditing rules for docker 
+
+Add the following statements to the audit rules file - `/etc/audit/rules.d/audit.rules`
+
+```
+-w /usr/bin/docker -p wa
+-w /var/lib/docker -p wa
+-w /etc/docker -p wa
+-w /lib/systemd/system/docker.service -p wa
+-w /lib/systemd/system/docker.socket -p wa
+-w /etc/default/docker -p wa
+-w /etc/docker/daemon.json -p wa
+-w /usr/bin/docker-containerd -p wa
+-w /usr/bin/docker-runc -p w
+```
+Restart Audit Daemon using below commands
+```
+$ sudo systemctl restart auditd
+ ```
+### B. Configuring Docker daemon
+
+Add the following statements to the docker daemon json file - `/etc/docker/daemon.json`
```

Comment:
Why are these changes required? What purpose do they serve for your image, because none of these seem required to me.